less/home-page.less

```diff
@@ -1,61 +1,98 @@
-      padding-right: 1%;
+  .main-header {
+        font-size: 2.5rem;
+       
+        img {
+            margin-bottom: 2%;
+        }
     }
+    .content-section {
+        margin: 30px 0;
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        
+        
+        .text-content {
+            width: 48%;
+            padding-right: 1%;
+        }
+        .img-content {
+            width: 48%;
+            padding-left: 1%;
+            
+            
+            img {
+                border-radius: 10px;
+            }
+          }
+          
+          @media @phone {
+              display: flex;
+              flex-direction: column;
+              flex-wrap: wrap;
+          }
+          .text-content {
+              width: 100%
+          }
+          
+          .img-content {
+              width: 100%
+          }
+          
 
-    .img-content {
-      width: 48%;
-      padding-left: 1%;
-
-      img {
-        border-radius: 10px;
-      }
-    }
-  }
 
-  .inverse-content {
-    padding-bottom: 30px;
-    border-bottom: 2px dashed @navigation-border;
+          @media @phone {
+            display: flex;
+            flex-direction: column;
+            flex-wrap: wrap;
+        }
 
-    .text-content {
-      padding-left: 1%;
-      padding-right: 0;
-    }
+      }
+                  .inverse-content {
+                      padding-bottom: 30px;
+                      border-bottom: 2px dashed @navigation-border;
+                      
+                      
+                      .text-content {
+                          padding-left: 1%;
+                          padding-right: 0;
+                      }
+                      .img-content {
+                          padding-right: 1%;
+                          padding-left: 0;
+                      }
+                  }
 
-    .img-content {
-      padding-right: 1%;
-      padding-left: 0;
-    }
-  }
 
-  .content-destination {
-    width: 75%;
-    margin: 0 auto 30px;
 
-    img {
-      border-radius: 10px;
-    }
-  }
+          .content-pick {
+            padding-top: 30px;
+            border-top: 2px dashed @navigation-border;
+            display: flex;
+            justify-content: space-between;
+          }
+          
+          @media @phone {
+              width: 100%;
+              margin-left: 5%;
+              padding-top: 30px;
+              border-top: 2px dashed @navigation-border;
+              display: flex;
+              flex-direction: column;
+              flex-wrap: wrap;
+          }
+      }
 
-  .content-pick {
-    padding-top: 30px;
-    border-top: 2px dashed @navigation-border;
-    display: flex;
-    justify-content: space-between;
+        .destination {
+          width: 30%;
+          margin-bottom: 30px;
+        }
+      
+    
 
-    .destination {
-      width: 30%;
+    @media @phone {
+      width: 100%;
       margin-bottom: 30px;
     }
-  }
-}// home
```

Comment:
awesome job at getting the media queries. the mobile design is missing some styles.